CONTRIBUTING.md

```diff
@@ -0,0 +1,247 @@
+
+- [Contributing to Plover](#contributing-to-plover)
+  - [Reporting Issues](#reporting-issues)
+    - [Ask for Help](#ask-for-help)
+    - [Search Open Issues](#search-open-issues)
+    - [Create a New Issue](#create-a-new-issue)
+      - [New Issue Template](#new-issue-template)
+  - [Contributing Code](#contributing-code)
+    - [Picking an Issue](#picking-an-issue)
+    - [Code Style](#code-style)
+    - [Commit Style](#commit-style)
+    - [Review Workflow](#review-workflow)
+    - [Labels](#labels)
+      - [Pull Request Review Flow](#pull-request-review-flow)
+      - [Issue Priority](#issue-priority)
+      - [Issue Category](#issue-category)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Contributing to Plover
+
+## Reporting Issues
+Run into a problem?
+
+### Ask for Help
+If you're having trouble using or configuring Plover,
+you might get more eyes on your problem
+and more immediate help
+by reaching out to the whole community:
+
+- [The Plover Google Group](https://groups.google.com/forum/#!forum/ploversteno)
+- [The Plover Discord Chat Server](https://discord.gg/0lQde43a6dGmAMp2)
+
+
+### Search Open Issues
+Start by searching the
+[list of open issues](https://github.com/openstenoproject/plover/issues).
+
+If you can find an issue that's the same as yours:
+
+- You might learn of a workaround from the discussion thread.
+- You might be able to provide the crucial detail needed to reproduce
+  and resolve the issue by adding a new comment with your details.
+
+### Create a New Issue
+You can
+[create a new issue](https://github.com/openstenoproject/plover/issues/new).
+You can help us by copying this template:
+
+#### New Issue Template
+```markdown
+Classification: (Pick one of: Bug, Crash/Hang/Data Loss, Performance, UI/Usability, Feature (New), Enhancement)
+
+Reproducibility: (Pick one of: Always, Sometimes, Rarely, Unable, I Didn't Try, Not Applicable - N/A applies for Feature and Enhancement requests)
+
+## Summary
+Summary of the problem.
+
+## Steps to Reproduce
+1. I do this
+2. I do that
+
+## Expected Results
+3. This should happen.
+
+## Actual Results
+3. Instead, this happens. :(
+
+## Version
+Plover version ???.
+(Click the "About…" button in the main Plover window to see the version
+number.)
+
+Installed via: (download from openstenoproject.org, Homebrew, apt-get, …).
+
+## Notes
+Describe any debugging steps you've taken yourself.
+
+If you've found a workaround, provide it here.
+
+## Configuration
+OS:
+
+- If it's OS X,
+  go to Finder, Apple menu, "About This Mac".
+  - Alternatively, if you're comfortable with the Terminal, run
+    `sw_vers` and paste the output.
+
+- If it's Linux,
+  provide information on your distro and any customization
+  relevant to Plover's operation (desktop environment, window manager, ???).
+  - uname -a output might also be useful.
+
+- If it's Windows,
+  (Ted/Benoit: Where should someone look for the Windows version?)
```

Comment:
@openstenoproject/developers I could use your help on filling out the "where to find the Windows version" section of this. :)